apps/astarte_trigger_engine/mix.exs

```diff
@@ -53,15 +53,17 @@ defmodule Astarte.TriggerEngine.Mixfile do
   defp astarte_required_modules("true") do
     [
       {:astarte_core, in_umbrella: true},
-      {:astarte_data_access, in_umbrella: true}
+      {:astarte_data_access, in_umbrella: true},
+      {:astarte_generators, in_umbrella: true}
```

Comment:
This is not needed, but probably none of the required modules is, so no problem in keeping it